src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -464,7 +464,8 @@ namespace OpenRCT2::Ui::Windows
             // Since the plugin may alter widget positions and sizes during an update event,
             // we need to force an update for all list view scrollbars
             WidgetIndex widgetIndex = 0;
-            for (auto widget = widgets; widget->type != WindowWidgetType::Empty; widget++)
+            for (auto widget = widgets; (widget->type != WindowWidgetType::Empty) && (widget->type != WindowWidgetType::Last);
+                 widget++)
```

Comment:
```suggestion
            for (auto widget = widgets; widget->type != WindowWidgetType::Last; widget++)
```

perhaps that was the original intent

Reply:
This is what it should be. When the widgets are created, it adds last to the end.